### ISSUE TITLE
Volumes broken when reading docker-compose file

### ISSUE THREAD

**Issue by AlexanderRichey** [aws/amazon-ecs-cli#661]
### Summary
ecs-cli suddenly cannot understand the `volumes` parameter of docker-compose files. I'm using this feature to share static files with an attached nginx container. I've tried changing my docker-compose to use `volumes_from` and I've also tried removing the `volumes` parameter from either of my containers, but the same error is thrown in all circumstances. When I remove the `volumes` parameter altogether, I get this error `InvalidParameterException: Container port 80 is used in more than one port mapping for container nginx`, which doesn't make any sense to me, given the configuration I'm using (see below).

### Description

* Which specific command was run?
```
ecs-cli compose --file $COMPOSE_FILE --project-name $ECS_PROJECT service up --timeout 10
```
* Which version of the CLI you are using?
v1.10.0
* Which version of Go are you using?
Using a precompiled binary on Linux
* What platform are you using to run ECS CLI commands? (E.g. Linux, MacOSX, Windows)
Linux

### Config files
```
version: '2'

services:
  application:
    image: "${...}.dkr.ecr.us-west-2.amazonaws.com/${...}"
    env_file: ...
    mem_limit: 805306368
    logging:
      driver: awslogs
      options:
        awslogs-region: us-west-2
        awslogs-group: ...
    command: ./docker/django/start.sh
    volumes:
      - static:/var/www/static
    ports:
      - "8000"
  nginx:
    image: "meural/nginx:latest"
    mem_limit: 134217728
    logging:
      driver: awslogs
      options:
        awslogs-region: us-west-2
        awslogs-group: ...
    ports:
      - "80"
      - "443"
    volumes:
      - static:/var/www/static
    links:
      - application
volumes:
  static:
```

### Expected Behavior
A new task definition is created and issued to the indicated cluster.
### Observed Behavior

```
ERRO[0000] Error registering task definition             error="ClientException: containerPath values aren't unique across mountPoints.\n\tstatus code: 400, request id: ceff4
308-e846-11e8-8105-a9eec4365095" family=...
ERRO[0000] Create task definition failed                 error="ClientException: containerPath values aren't unique across mountPoints.\n\tstatus code: 400, request id: ceff4
308-e846-11e8-8105-a9eec4365095"
FATA[0000] ClientException: containerPath values aren't unique across mountPoints.
        status code: 400, request id: ceff4308-e846-11e8-8105-a9eec4365095
```

**Comment by allisaurus** [comment]
Hi @AlexanderRichey - I was able to register a task definition with the above volume configuration. Mounting a named volume ("`static`") into 2 different containers at the same location ("`/var/www/static`") should be fine. 

The error message indicates 2 volumes were trying to mount to the same location on one container. Is there perhaps a 2nd compose file with `nginx` being erroneously read in [as an override](https://docs.docker.com/compose/extends/#understanding-multiple-compose-files)? (either in the working dir or the content of your `$COMPOSE_FILE` env var.) This would also explain the msg re: port 80 being used more than once in that container.

Also, to confirm, you're not using an `ecs-params` file to set additional networking or other settings, correct?

Thanks!

**Comment by AlexanderRichey** [comment]
Thanks for your response @allisaurus! The problem was caused by the `$COMPOSE_FILE` env var. I didn't realize that the ecs-cli searches for this variable and automatically loads that configuration. I solved the problem by renaming `$COMPOSE_FILE` to `$ECS_COMPOSE_FILE`.

It seems like the underlying issue was that the command `ecs-cli compose --file $COMPOSE_FILE` actually tried to load the configuration twice - once from the magically named env var and again from the command line argument. Thanks for your help!
